typescript/src/deposit.ts

```diff
@@ -344,3 +382,28 @@ export async function revealDeposit(
 
   await bridge.revealDeposit(depositTx, utxo.outputIndex, deposit)
 }
+
+/**
+ * Gets a revealed deposit from the bridge.
+ * @param utxo Deposit UTXO of the revealed deposit
+ * @param bridge Handle to the Bridge on-chain contract
+ * @returns Revealed deposit data.
+ */
+export async function getRevealedDeposit(
+  utxo: UnspentTransactionOutput,
+  bridge: Bridge
+): Promise<RevealedDeposit> {
+  return bridge.deposits(utxo.transactionHash, utxo.outputIndex)
+}
+
+/**
+ * Suggests a wallet that should be used as the deposit target at the given moment.
+ * @param bridge Handle to the Bridge on-chain contract.
+ * @returns Compressed (33 bytes long with 02 or 03 prefix) public key of
+ *          the wallet.
```

Comment:
Maybe we should specify in JSDoc also that it returns a promise.

```suggestion
 * @returns Promise object represents compressed (33 bytes long with 02 or 03 prefix) public key of
 *          the wallet.
```